packages/react-components/src/components/VideoGallery/RemoteScreenShare.tsx

```diff
@@ -67,6 +71,22 @@ export const RemoteScreenShare = React.memo(
           participant: displayName
         })
       : '';
+    /* @conditional-compile-remove(ppt-live) */
```

Comment:
Add same TODO here
`// TODO: need to remove after contentSharingRole avaible in WebCalling SDK.`